cmake/modules/BuildFIO.cmake

```diff
@@ -0,0 +1,15 @@
+function(build_fio)
+  # we use an external project and copy the sources to bin directory to ensure
+  # that object files are built outside of the source tree.
+  include(ExternalProject)
+  ExternalProject_Add(fio_ext
+    DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
+    GIT_REPOSITORY "https://github.com/axboe/fio.git"
+    GIT_TAG "e09d68c8da4ab91397490577454de928106651f5"
+    SOURCE_DIR ${CMAKE_BINARY_DIR}/src/fio
+    BINARY_DIR ${CMAKE_BINARY_DIR}/src/fio
+    CONFIGURE_COMMAND ${CMAKE_BINARY_DIR}/src/fio/configure
+    BUILD_COMMAND $(MAKE) fio EXTFLAGS=-Wno-format-truncation
+    COMMAND cp ${CMAKE_BINARY_DIR}/src/fio/fio ${CMAKE_BINARY_DIR}/bin
```

Comment:
nit, this *is* the `INSTALL_COMMAND` =)

Reply:
also, could put
```cmake
INSTALL_COMMAND install <BINARY_DIR>/fio ${CMAKE_BINARY_DIR}/bin
```
instead,  to use `INSTALL_COMMAND`, and to use the `BINARY_DIR`.